### PR TITLE
Removed promise & fetch shim from being added to build since they are now polyfilled

### PR DESCRIPTION
**Pull request by yomexzo**
👀 @richgilbank @minasmart @tessalt

### CODE REVIEW

File: build-lib/lib.js [hunk 1]
```diff
@@ -18,12 +18,7 @@ const Versioner = require('./util/versioner');
 function sourceTree(pathConfig, moduleType) {
   const lib = babelTranspiler(pathConfig.lib, babelConfig(pkg.name, moduleType));
 
-  const shims = babelTranspiler(
-    funnel(pathConfig.shims, { include: ['fetch.js', 'promise.js'] }),
-    babelConfig(null, moduleType)
-  );
-
-  return mergeTrees([lib, shims]);
```
Comment (minasmart) [inline build-lib/lib.js]: You've also removed the fetch polyfill. Was that intentional? Is it used anywhere?

Comment (yomexzo) [inline build-lib/lib.js]: yeah, `fetch.js` does not exist. I think you removed it when you moved it to being polyfilled way back.

Comment (minasmart) [inline build-lib/lib.js]: Maybe just adjust the PR title, so we know the full effects then.

Comment (minasmart) [inline build-lib/lib.js]: And remove `fetch.js` from the shims directory.

Comment (yomexzo) [inline build-lib/lib.js]: `fetch.js` does not exist in the shims directory. You removed it already.

Comment (yomexzo) [inline build-lib/lib.js]: i think you did rather. But it's not there. Editing the title now.

**Comment by yomexzo** [comment]
The Promise shim is not needed any more since `Promise` is being polyfilled [here](https://github.com/Shopify/js-buy-sdk/blob/master/build-lib/polyfills.js#L17). Fetch shim is also being polyfilled.

File: build-lib/lib.js [hunk 2]
```diff
@@ -18,12 +18,7 @@ const Versioner = require('./util/versioner');
...
+  return mergeTrees([lib]);
```
Comment (minasmart) [inline build-lib/lib.js]: You can just return lib here, since it's one tree.

Comment (yomexzo) [inline build-lib/lib.js]: true. done. thanks.

**Comment by minasmart** [comment]
Can you rebase and get this in?

**Pull request merged**